tests/data/claypit/tests/_support/ScenarioGuy.php

```diff
@@ -20,10 +20,45 @@ class ScenarioGuy extends \Codeception\Actor
 {
     use _generated\ScenarioGuyActions;
 
-    function seeCodeCoverageFilesArePresent()
+    public function seeCodeCoverageFilesArePresent()
     {
         $this->seeFileFound('c3.php');
         $this->seeFileFound('composer.json');
         $this->seeInThisFile('codeception/c3');
     }
+
+    /**
+     * @Given I have terminal opened
```

Comment:
Is it mandatory to have a Given line in the feature file?
This one is so useless that it would be better to skip it.